feat: adding grafana examples (#255)
mergestat/fuse: README.md

You can manage a single PAT for your instance in the `Settings` area of the management UI.

> Now if you visit `http://localhost:3000/` you should be able to access Grafana where you can see example dashboards using the data from Fuse

Reviewer: ```suggestion
Now you can also visit `http://localhost:3000/` to access a local [Grafana](https://grafana.com/) instance with example dashboards using the data from Fuse 🎉 :
```
